zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/RequestMapper.java

```diff
@@ -182,6 +196,28 @@ private static Optional<ProblemDetail> validateJobActivationRequest(
             HttpStatus.BAD_REQUEST, String.join(". ", violations), INVALID_ARGUMENT.name()));
   }
 
+  private static Optional<ProblemDetail> validateJobFailRequest(
+      final JobFailRequest failRequest
+  ) {
+    final List<String> violations = new ArrayList<>();
+    if (failRequest != null) {
+      // retries can't be less than 0
+      if (failRequest.getRetries() < 0) {
```

Comment:
This is an assumption that I made, since I'm not able to find any documentation about it. But I think it make sense to not allow users to provide strange values

Reply:
Well, the engine doesn't care much, it creates an incident when it's 0 or lower: https://github.com/camunda/camunda/blob/main/zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/job/JobFailProcessor.java#L135-L137
That being said, I have no idea what people will want to use here, not sure if we need to be too strict here.

Reply:
You are suggesting to remove the validation at all ?

Reply:
Yep, I think we can remove it. The engine can apparently handle it and we don't know if users are actually maybe more comfortable or used to handing in -1, for example 🤷